### ISSUE TITLE
ACL examples YAML error

### ISSUE THREAD

**Issue by yekangming** [traefik/mesh#778]
## Bug Report

For the official document, but there was an unexpected error. 
The error was caused by the wrong information configured by yaml. Port should be a number instead of a string.

### What did you do?

```yaml
kind: TrafficTarget
apiVersion: access.smi-spec.io/v1alpha2
metadata:
  name: whatever
  namespace: whoami
spec:
  destination:
    kind: ServiceAccount
    name: whoami-server
    namespace: whoami
    port: "80"
  rules:
    - kind: HTTPRouteGroup
      name: http-everything
      matches:
        - everything
  sources:
    - kind: ServiceAccount
      name: whoami-client
      namespace: whoami
```

### Output of controller log: (_What version of Traefik Mesh are you using?_)

`The TrafficTarget "whatever" is invalid: spec.destination.port: Invalid value: "string": spec.destination.port in body must be of type number: "string"`

### What is your environment & configuration (arguments, provider, platform, ...)?

https://doc.traefik.io/traefik-mesh/examples/#acl-example

**Comment by yekangming** [comment]
Correct
```json
kind: TrafficTarget
apiVersion: access.smi-spec.io/v1alpha2
metadata:
  name: whatever
  namespace: whoami
spec:
  destination:
    kind: ServiceAccount
    name: whoami-server
    namespace: whoami
    port: 80
  rules:
    - kind: HTTPRouteGroup
      name: http-everything
      matches:
        - everything
  sources:
    - kind: ServiceAccount
      name: whoami-client
      namespace: whoami
```

**Comment by traefiker** [comment]
Closed by #780.
